app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/ThreadController.kt

```diff
@@ -108,13 +107,14 @@ class ThreadController @Inject constructor(
                 // The Search only returns Messages from TRASH if we explicitly selected this folder,
                 // which is the reason why we can compute the `isTrashed` value so loosely.
                 remoteMessage.initLocalValues(
-                    MessageInitialState(
-                        isFullyDownloaded = localMessage?.isFullyDownloaded() ?: false,
-                        isTrashed = filterFolder?.role == FolderRole.TRASH,
-                        isFromSearch = localMessage == null,
-                        draftLocalUuid = localMessage?.draftLocalUuid,
-                    ),
-                    latestCalendarEventResponse = null,
+                    isFullyDownloaded = localMessage?.isFullyDownloaded() ?: false,
```

Comment:
Same question here about `isFullyDownloaded()` when we have a localMessage to keep local values from